src/raw/mmap.rs

```diff
@@ -50,6 +57,23 @@ impl MmapReadOnly {
     }
 }
 
+impl Clone for MmapReadOnly {
+
+    fn clone(&self) -> MmapReadOnly {
+        MmapReadOnly{
+            map: self.map.clone(),
+            offset: self.offset,
+            len: self.len,
+        }
+    }
+
+    fn clone_from(&mut self, source: &Self) {
```

Comment:
I think we can leave this specialization out. The default implementation of `clone_from` should be good enough. :-)